src/main/kotlin/app/cash/licensee/pluginExtension.kt

```diff
@@ -29,6 +29,13 @@ import org.gradle.api.provider.SetProperty
 
 @Suppress("unused") // Public API for Gradle build scripts.
 interface LicenseeExtension {
+
+  /**
+   * Enable asset generation. Will copy licensee report to
+   * android asset directory making it available as 'androidAssetFileName'
+   */
+  fun enableAndroidAssetGeneration(enabled: Boolean)
```

Comment:
Using "enable" as a verb but also taking a boolean can be viewed as either redundant or potentially contradictory. I think we should probably just expose a `Property<Boolean>` with a name like `bundleAndroidAsset` (it's not really toggling generation, just the bundling step).

Reply:
agreed